extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -210,6 +231,10 @@ export class AmpA4A extends AMP.BaseElement {
         this.preconnect.url(p, true);
       });
     }
+    // TODO(tdrl): Temporary, while we're verifying whether SafeFrame is an
+    // acceptable solution to the 'Safari on iOS doesn't fetch iframe src from
+    // cache' issue.  See https://github.com/ampproject/amphtml/issues/5614
+    this.preconnect.preload(SAFEFRAME_IMPL_PATH);
```

Comment:
Preload nominally does preconnect as well, but happy enough to change it to preconnect.  Done.